code/model/DMSDocument.php

```diff
@@ -914,6 +914,13 @@ public function getCMSFields()
         $fields->add(new TextField('Title', 'Title'));
         $fields->add(new TextareaField('Description', 'Description'));
 
+        $downloadBehaviorSource = array(
+            'default' => 'Default ('.DMSDocument_Controller::$downloadBehavior.')',
```

Comment:
I'm not sure if having an option `default` is the best here. Perhaps the option in the dropdown should just be selected by default? Because if a dev changes the default behaviour at one point then all the docs will change their behaviour which may not have been intended by the CMS user when they selected "default".